layout/_partial/post/gitment.ejs

```diff
@@ -0,0 +1 @@
+<div id="gcomment"></div>
```

Comment:
I think `gitment` or `gitment-comments` would be more appropriate

Reply:
Changed as requested. :)